lib/src/HttpClientImpl.h

```diff
@@ -33,9 +33,11 @@ class HttpClientImpl : public HttpClient,
     HttpClientImpl(trantor::EventLoop *loop,
                    const trantor::InetAddress &addr,
                    bool useSSL = false,
+                   bool validateCert = true,
```

Comment:
The order of validateCert and useOldTLS is conflicted with implementations and callers. Please make them consistent.

Reply:
Should be solved.